packages/menu/src/MenuItem/MenuItem.js

```diff
@@ -0,0 +1,71 @@
+import React, { memo } from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { Button, Flex, getPaletteColor } from 'pcln-design-system'
+import { Check as CheckIcon } from 'pcln-icons'
+
+const MenuButton = styled(Button)`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  border: 2px solid transparent;
+  border-radius: 6px;
+  color: ${getPaletteColor('text.dark')};
+  outline: none;
+  padding: 12px;
+
+  &:focus {
+    border: 2px solid ${getPaletteColor('base')};
+  }
+
+  &:hover {
+    background-color: ${getPaletteColor('base')};
+  }
+
+  ${(props) =>
+    props.selected
+      ? `background-color: ${getPaletteColor('background.base')(props)};`
+      : `background-color: ${getPaletteColor('background.lightest')(props)};`}
+`
+
+const MenuItem = React.forwardRef(function MenuItem(
```

Comment:
This line is so weird feels like it should complain about `const MenuItem` and `function MenuItem` sharing the same name.

Reply:
You think it would be it doesn't 🤷‍♂️